CONTRIBUTING.md

```diff
@@ -44,42 +59,62 @@ by [Vincent Driessen](https://nvie.com/posts/a-successful-git-branching-model/):
     ```bash
     git checkout -b feature/myfeature
     ```
-    It is best to merge one's changes *as fast as possible* (i.e. do not wait for 6 months) to avoid merging conflicts
-3. Open [Protégé](https://protege.stanford.edu/) and work on the ontology. If you haven't already, make sure you change your protégé settings to use [numeric identifiers](https://github.com/OpenEnergyPlatform/ontology/wiki/Numerical-Identifiers).
+    It is best to merge one's changes *as fast as possible* (i.e. do not wait for 2 months) to avoid merging conflicts
+3. 📙 or 📗 Open [Protégé](https://protege.stanford.edu/) or a text editor and work on the ontology. If you haven't already, make sure you change your protégé settings to use [numeric identifiers](https://github.com/OpenEnergyPlatform/ontology/wiki/Numerical-Identifiers).
```

Comment:
Somewhere here it has to be mentioned that the correct oeo module file has to be selected before adding new classes or relations.

Reply:
Additionally one could mention here (maybe with the assistance of a screenshot) that for most cases one has to work in Protégé in the yellow tabs (_Entities_ and _Classes_). As we saw last week from the practical examples one major hurdle was to find in Protégé the correct place to work.